plugins/lookup/aws_secret.py

```diff
@@ -19,12 +19,16 @@
 description:
   - Look up secrets stored in AWS Secrets Manager provided the caller
     has the appropriate permissions to read the secret.
-  - Lookup is based on the secret's `Name` value.
-  - Optional parameters can be passed into this lookup; `version_id` and `version_stage`
+  - Lookup is based on the secret's I(Name) value.
+  - Optional parameters can be passed into this lookup; I(version_id) and I(version_stage)
 options:
   _terms:
     description: Name of the secret to look up in AWS Secrets Manager.
     required: True
+  bypath:
+    description: A boolean to indicate whether the parameter is provided as a hierarchy.
+    default: false
+    type: bool
```

Comment:
```suggestion
    type: bool
    version_added: 1.4.0
```

Reply:
In addition to adding the `version_added`, I also changed the type to `boolean`  for consistency.